res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1986,45 +1986,35 @@ NumarkMixtrack3.bpmTap = function(channel, control, value, status, group) {
 NumarkMixtrack3.EQKnob = function(channel, control, value, status, group) {
     var deck = NumarkMixtrack3.deckFromGroup(group);
     var decknum = deck.decknum;
+    var focusedEffect = engine.getValue("[EffectRack1_EffectUnit" + decknum + "]", "focused_effect");
```

Comment:
Make sure you set show_focus to 1 for every EffectUnit or the focus won't be shown on screen.